pkg/didcomm/common/messenger/messenger.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package messenger
```

Comment:
I assume this is a new package since these definitions are used beyond services. Correct?

- Shouldn't this package be more generic (e.g., message / msg).
- Shouldn't the DIDCommMsg interface be moved into this same package (in your next PR)?